content/en/logs/guide/ease-troubleshooting-with-cross-product-correlation.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Ease troubleshooting with cross product correlation
+kind: guide
+---
+
+## Overview
+
+[Unified service tagging][1] permits high level correlation capabilities. However, there are times when the starting point of your investigation is a single log, trace, view or Synthetic test. Correlating logs, traces, and views with other data gives context to help estimate business impact and find the root cause of an issue in a few clicks.
+
+{{< img src="logs/guide/ease-troubleshooting-with-cross-product-correlation/full-stack-cover.png" alt="Full stack correlation" style="width:80%;" >}}
+
+This guide walks you through the steps to correlate your full stack data:
+
+1. [Correlate server-side logs with traces](#correlate-server-side-logs-with-traces)
+   * [Correlate application logs](#correlate-application-logs)
+   * [Correlate proxy logs](#correlate-proxy-logs)
+   * [Correlate database logs](#correlate-database-logs)
+2. [Correlate frontend products](#correlate-frontend-products)
+   * [Correlate browser logs with RUM](#correlate-browser-logs-with-rum)
+3. [Correlate user experience with server behavior](#correlate-user-experience-with-server-behavior)
+   * [Correlate RUM views with traces](#correlate-rum-views-with-traces)
+   * [Leverage trace correlation to troubleshoot Synthetic tests](#leverage-trace-correlation-to-troubleshoot-synthetic-tests)
+
+**Note**: Depending on your use case, you may skip certain steps below. Steps that are dependent on others are explicitly stated.
+
+## Correlate server-side logs with traces
+
+When your users are encountering errors or high latency in your application, drilling down to view the logs from a problematic request can reveal exactly what went wrong. By pulling together all the logs pertaining to a given request, you can see in rich detail how it was handled from beginning to end so you can quickly diagnose the issue.
+
+Correlating your logs also eases [aggressive sampling strategy without losing entity-level consistency][2] with the use of attributes like `trace_id` and `session_id`.
```

Comment:
```suggestion
Correlating your logs with traces also eases [aggressive sampling strategy without losing entity-level consistency][2] with the use of attributes like `trace_id` and `session_id`.
```

Reply:
`Aggressive sampling strategy without losing entity-level consistency` is a benefit of log correlation in general (as a matter of fact, the `session_id` example will come from RUM correlation). I wrote it here because it is the main log correlation, but it is not only with traces

Reply:
What do you think of duplicating this sentence? Would it be better

In https://docs-staging.datadoghq.com/gturbat/new-guide-log-correlation/logs/guide/ease-troubleshooting-with-cross-product-correlation/?tab=nginx#correlate-server-side-logs-with-traces:

> Correlating your logs with traces also eases [aggressive sampling strategy without losing entity-level consistency][2] with the use `trace_id`.

In https://docs-staging.datadoghq.com/gturbat/new-guide-log-correlation/logs/guide/ease-troubleshooting-with-cross-product-correlation#why-3:

> Correlating your browser logs with RUM also eases [aggressive sampling strategy without losing entity-level consistency][2] with the use `session_id`.

Reply:
Ah, I see. I misunderstood the relevancy to the overall subject of correlation versus to the header since it's under `Correlate server-side logs with traces`. We can duplicate in both areas I think would be best, just to avoid confusion.

Reply:
Done in https://github.com/DataDog/documentation/pull/10296/commits/8eecd29c793714c43f68c7f42e635d26e0c4c916